src/scripts/servers.js

```diff
@@ -65,10 +65,13 @@ class Servers extends EventEmitter {
 
         // Load server info from server config file
         if (Object.keys(hosts).length === 0) {
-            const serverFileName = 'servers.json';
-            const userDataDir = jetpack.cwd(remote.app.getPath('userData'));
+            const pathToServerJson = jetpack.path(
```

Comment:
Now we check the user APPDATA folder first, then the system installation folder (for /allusers) and then the packaged file (if we do a branded package).